src/Java.Runtime.Environment/Java.Interop/JreRuntime.cs

```diff
@@ -39,11 +39,7 @@ public class JreRuntimeOptions : JniRuntime.CreationOptions {
 		public JreRuntimeOptions ()
 		{
 			JniVersion  = JniVersion.v1_2;
-			ClassPath   = new Collection<string> () {
-				Path.Combine (
-					Path.GetDirectoryName (typeof (JreRuntimeOptions).Assembly.Location) ?? throw new NotSupportedException (),
-					"java-interop.jar"),
-			};
+			ClassPath   = new Collection<string> ();
```

Comment:
What is `Collection<string>`? I've never heard of this? It would be an API break to change it, so probably can't be a `List<string>`.

Reply:
[`Collection<T>`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.collection-1?view=net-7.0).  It's existed since .NET Framework 2.0, *and* is the recommended collection type in the [.NET Framework Design Guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections):

> ✔️ DO use Collection<T> or a subclass of `Collection<T>` for properties or return values representing read/write collections.

This isn't an API break because the type hasn't changed at all, and -- again -- `Collection<T>` is in fact the preferred collection type here.

Reply:
TIL, `List` vs `Collection`: https://stackoverflow.com/a/271842

Reply:
Additionally, we've never "shipped" `Java.Runtime.Environment.dll`, so I'm fine witih making API breaks here if warranted.